feat: RPC API server listens on loopback by default (#142)
libp2p/js-libp2p-amino-dht-bootstrapper: README.md

Options:
      --config <CONFIG>              Path to IPFS config file (required)
      --metrics-path <METRICS_PATH>  Metric endpoint path [default: /metrics]
      --metrics-port <PORT>          Metric endpoint path [default: 8888]
      --enable-kademlia              Whether to run the libp2p Kademlia protocol and join the IPFS DHT
      --enable-autonat               Whether to run the libp2p Autonat protocol
      --api-port <PORT>              Port to serve the RPC API [default: 8899]
      --api-host <HOST>              Host to serve the RPC API on [default: 127.0.0.1]
  -h, --help                         Print help
```

### RPC API

To make a request via CURL, you can use the following command:

```sh
# run garbage collection
$ curl http://${HOST}:${RPC_PORT}/api/v0/nodejs/gc

# execute a heapdump
$ curl http://${HOST}:${RPC_PORT}/api/v0/nodejs/heapdump
```

> Note that the RPC API server only listens on the loopback interface (127.0.0.1) by default. If you change the `api-host` option, you should ensure that the RPC API server is only being used for development purposed, or not accessible publicly.

Author: ```suggestion
Please note that the RPC API server only listens on the loopback interface (127.0.0.1) by default. If you decide to change the `api-host` option, please make sure that the RPC API server is only used for development purposes and is not accessible publicly.
```
